supervision/dataset/formats/coco.py

```diff
@@ -159,16 +173,29 @@ def detections_to_coco_annotations(
     return coco_annotations, annotation_id
 
 
+def get_coco_class_index_mapping(annotations_path: str) -> Dict[int, int]:
+    coco_data = read_json_file(annotations_path)
+    classes = coco_categories_to_classes(coco_categories=coco_data["categories"])
+    class_mapping = build_coco_class_index_mapping(
+        coco_categories=coco_data["categories"], target_classes=classes
+    )
+    inv_class_mapping = {v: k for k, v in class_mapping.items()}
+    return inv_class_mapping
+
+
 def load_coco_annotations(
     images_directory_path: str,
     annotations_path: str,
     force_masks: bool = False,
+    # use_iscrowd: bool = True,
```

Comment:
I noticed that in [this commit](https://github.com/roboflow/supervision/pull/1834/commits/cff974555e9d5f2245d9fa83940abcc114a84d3f#diff-64070d7a771b2a8229f1551983f12235d058b6af2fce4a834a626fd13eced843L566), you removed use_iscrowd from the public API of DetectionDataset.from_coco. I agree with this change. However, I think we should keep the use_iscrowd parameter available at the load_coco_annotations level.